x-pack/plugins/security_solution/public/detections/components/value_lists_management_modal/modal.test.tsx

```diff
@@ -47,15 +74,60 @@ describe('ValueListsModal', () => {
     container.unmount();
   });
 
-  it('renders ValueListsForm and ValueListsTable', () => {
+  it('renders ValueListsForm and an EuiTable', () => {
     const container = mount(
       <TestProviders>
         <ValueListsModal showModal={true} onClose={jest.fn()} />
       </TestProviders>
     );
 
     expect(container.find('ValueListsForm')).toHaveLength(1);
-    expect(container.find('ValueListsTable')).toHaveLength(1);
+    expect(container.find('EuiBasicTable')).toHaveLength(1);
     container.unmount();
   });
+
+  describe('modal table actions', () => {
+    it('calls exportList when export is clicked', () => {
+      const container = mount(
+        <TestProviders>
+          <ValueListsModal showModal={true} onClose={jest.fn()} />
+        </TestProviders>
+      );
+
+      act(() => {
+        container
+          .find('tbody tr')
```

Comment:
Would be nice if we didn't have to use non data-test-subj but understand if we do.

Reply:
The scoping is likely unnecessary, but I found it more legible to do it this way: "find the first row and click its export button" vs "click the first export button." If the build fails I'll update these.

Reply:
Yeah I tend to think if the container's implementation ever changes then we will have to update tests. Nothing wrong with it if you want to keep it.